### DISCUSSION TITLE
Welcome to SQLite Cloud Support Center

### DISCUSSION THREAD

**Discussion by marcobambini** [sqlitecloud/discussions#1] [Announcements]
We're excited to announce the launch of our GitHub Support Center – your go-to hub for all things support-related! Whether you're encountering issues, have questions, or want to report bugs, this is the place to be.

At SQLite Cloud, we believe in fostering a collaborative and transparent environment, and our GitHub Support Center is designed to facilitate seamless communication between our team and the incredible users like you. Here's what you can do here:

- **Get Support**: If you're facing any challenges or need assistance, feel free to reach out. Our dedicated support team is here to help you navigate any hurdles.
- **Report Bugs**: If you come across any bugs or unexpected behavior, please report them here. Your feedback is invaluable in helping us improve and deliver a more robust product.
- **Ask Questions**: Have a burning question or want clarification on a feature? Ask away! Our community, as well as our experts, are here to share insights and help you make the most of SQLite Cloud.

We're committed to continuously enhancing your experience with SQLite Cloud, and your input plays a crucial role in shaping the future of our platform.

Thank you for being part of the SQLite Cloud community.
Let's build something great together!

**Comment by SACPDG** [comment]
earlier when the app tried to connect to db and the db was in sleep it would wake up automatically, is it an update that we need to wake it up manually

**Reply by andinux** [reply]
You’re right, the behavior has changed. For free projects, databases no longer wake up automatically when accessed. They now require a manual restart to be reactivated.
